aries-site/src/examples/components/checkbox/CheckBoxMultExample.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+import { Box, CheckBox, Text } from 'grommet';
+
+const items = ['C++', 'JavaScript', 'Python'];
+
+export const CheckBoxMultExample = () => {
+  const [checked, setChecked] = useState(['JavaScript']);
+
+  const onCheck = (event, item) => {
+    if (event.target.checked) {
+      setChecked([...checked, item]);
+    } else {
+      setChecked(checked.filter(el => el !== item));
+    }
+  };
+
+  return (
+      <Box gap="medium">
+        <Text>Select prefered Language</Text>
```

Comment:
We should put this in a FormField like the other examples. Unless there is a reason we're not doing that here.

Reply:
I thought we were having one without `formfield `to still explain that this is possible but you should still provide a label ? 

Unless I heard wrong then I can put back `formfield`